tests/hcaptcha.spec.js

```diff
@@ -120,6 +126,27 @@ describe("hCaptcha", () => {
         expect(window.hcaptcha.remove.mock.calls[0][0]).toBe(MOCK_WIDGET_ID);
     });
 
+    it("can get Response", () => {
+        expect(window.hcaptcha.getResponse.mock.calls.length).toBe(0);
+        instance.getResponse();
+        expect(window.hcaptcha.getResponse.mock.calls.length).toBe(1);
+        expect(window.hcaptcha.getResponse.mock.calls[0][0]).toBe(MOCK_WIDGET_ID);
+    });
+
+    it("can get RespKey", () => {
+        expect(window.hcaptcha.getRespKey.mock.calls.length).toBe(0);
+        instance.getRespKey();
```

Comment:
Lacking check for the return value of  `instance.getRespKey()`.

```js
const res = instance.getRespKey();
expect(res).toBe(MOCK_EKEY);
```

Reply:
Done https://github.com/hCaptcha/react-hcaptcha/pull/122/commits/09a2cab4b5e9dae2f5543b4e5839100ad0618c5a